test/base/__init__.py

```diff
@@ -100,3 +113,114 @@ def forNode(self):
             'expires': self.expires,
             'scopes': self.scopes
         }
+
+
+class FakeGenerated(object):
```

Comment:
Sure. https://github.com/escapewindow/taskcluster-client.py/commit/b5075a822eb16b8085c4293898407b11ac74d896

I think an earlier revision of `FakeGenerated` also overrode several other methods with non-mock code, until I rewrote things and didn't need them anymore.  Easy enough to replace with a function now.